specifications/cdsc-wg1-01.md

```diff
@@ -1 +1,292 @@
+
+### 4.1. Coverage Object Format <a id="coverage-object-format" href="#coverage-object-format" class="permalink">🔗</a>
+
+Coverage objects are formatted as JSON objects and contain the following named values. This object serves mostly as an object wrapper around a list of [coverage entries](#coverage-entry-format), so that the list of coverage entries can be paginated by the Server if too long for one response.
+
+* `coverage_entries` - _Array[[CoverageEntry](#coverage-entry-format)]_ - (REQUIRED) A list of segments this Server covers
+* `next` - _`null` or URL_ - (REQUIRED) Link to the next page of coverage entries (`null` if no more subsequent pages)
+* `previous` - _`null` or URL_ - (REQUIRED) Link to the previous page of coverage entries (`null` if no more prior pages)
+
+### 4.2. Coverage Entry Format <a id="coverage-entry-format" href="#coverage-entry-format" class="permalink">🔗</a>
+
+Coverage entries are formatted as JSON objects and contain named values, some of which are required and some of which are optional. The following values are included in the default list available in coverage entries.
+
+* `id` - _string_ - (REQUIRED) A unique identifier for the coverage entry
+* `type` - _[CoverageEntryType](#coverage-entry-types)_ - (REQUIRED) Which Coverage Entry Type this coverage entry is
+* `updated` - _ISO8601 datetime_ - (REQUIRED) When this coverage entry was last modified
+* `name` - _string_ - (REQUIRED) A human readable name for the coverage area or category (e.g. "Upstate New York")
+* `description` - _string_ - (OPTIONAL) A brief description the coverage area or category (e.g. "Most of the northern half of New York state")
+* `map_resource` - _URL_ - (OPTIONAL) Link to a map of the coverage territory
+* `map_content_type` - _MIME type_ - (OPTIONAL) Content-Type format of the `map_resource`
+* `geojson_resource` - _URL_ - (OPTIONAL) Link to a GeoJSON object mapping the coverage territory
+
+When `type` is a geographic entry type, one or both of `map_resource` and `geojson_resource` MUST be provided. If `map_resource` is provided, then `map_content_type` MUST also be provided.
+
+### 4.3. Coverage Entry Types <a id="coverage-entry-types" href="#coverage-entry-types" class="permalink">🔗</a>
+
+Coverage entry types describe what geographic territory is being detailed in the coverage entry. Many utilities have different geographic coverage for their different service types (e.g. their electric service territory is different from their natural gas service territory), so Servers may include multiple coverage entries for different types of service without having to publish multiple metadata endpoints.
+
+The following list of strings are an enumerated set of coverage entry types that are valid `type` values in the coverage entry object.
```

Comment:
From @jhughes22 ([on May 25, 2023](https://github.com/carbon-data-specification/Customer-Data/pull/23#discussion_r1205771159)):

---

A balancing authority/ISO/TSO may have a coverage area that doesn't quite match any of these type descriptions. Something to consider if power systems related data is worked into the metadata advertising. The bit about extensions in section 6 may cover this though

---

Reply:
@jhughes22 Thanks for the feedback! I've moved infrastructure types to be in individual coverage entries and added `transmission_service_operator` and other types of infrastructure values to the most recent updates: https://daniel-utilityapi.github.io/Connectivity/specs/cdsc-wg1-01/#infrastructure-types